From: Julien Cristau <julien.cristau@ens-lyon.org>
Date: Sun, 7 Jun 2009 16:55:39 +0200
Subject: [PATCH] Disable svgz

librsvg doesn't have a rsvg-gz.h header anymore, and
rsvg_handle_new_gz isn't necessary (svgz files are detected
automatically). So don't build a binding for this function.
---
 config.make.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/config.make.in b/config.make.in
index e689bda..d5487d7 100644
--- a/config.make.in
+++ b/config.make.in
@@ -19,7 +19,7 @@ EXE=@EXE@
 USE_GL=@USE_GTKGL@
 USE_GLADE=@USE_GLADE@
 USE_RSVG=@USE_RSVG@
-HAVE_SVGZ=@HAVE_SVGZ@
+HAVE_SVGZ=
 USE_GNOMECANVAS=@USE_GNOMECANVAS@
 USE_GNOMEUI=@USE_GNOMEUI@
 USE_PANEL=@USE_PANEL@
-- 
